### ISSUE TITLE
Remove jQuery dependency

### ISSUE THREAD

**Issue by ghost** [lucidlemon/paradeiser#3]
I really like how Paradeiser looks and works. Thank you! :)

My biggest suggestion is to remove the jQuery "baggage" — consider the following (quoted from the You Might Not Need jQuery site):

> If you're developing a library (...) please take a moment to consider
> if you actually need jQuery as a dependency. Maybe you can include a few lines
> of utility code, and forgo the requirement. If you're only targeting more modern browsers,
> you might not need anything more than what the browser ships with.

Main advantages:
- **Users wait less time** for your pages to download -- Remember many people are on **mobile connections.**
- You use commands that are **touching the DOM directly**. jQuery is a wrapper around the DOM and, as such, many times makes it less intuitive to accomplish some things. **Know the DOM, you'll be better off this way.** :)
- **Simpler setup.** From all the JS libraries I've used, jQuery (being by far the most complex) was the only one that choked on my build system's **concatenate & uglify step.** My build system had to be much more complex just because of jQuery... Removed it. Done!

Some resources that are helpful for switching from jQuery to native JavaScript:
- [You Might Not Need jQuery](http://youmightnotneedjquery.com/)
- [jQuery considered harmful - Lea Verou](http://lea.verou.me/2015/04/jquery-considered-harmful/)

There's also this one (a bit more advanced):
- [Ditching jQuery - Go Make Things](http://gomakethings.com/ditching-jquery/)

Mmm... Should I just say it? — **"No jQuery" is the direction hipsters are taking.**  :D

The people in-the-know are now developing libraries without the jQuery requirement...

When feasible, a library should stand on its own feet — this way it is (more) future-proof, less dependent on other libraries, easier to get started with, and more flexible to be integrated in systems which don't use jQuery.

**jQuery was created for a time when the DOM was hard.** The DOM is pretty easy, now. ;)

**Comment by lucidlemon** [comment]
Thanks a lot for your feedback.

In fact, removing jQuery is already on my agenda.
Furthermore I even thought about removing JS entirely to just work with the css `:target` selector, but I figured out that having the grey box beneath the menu might be a bit tricky then.

The truth is, getting into... "more serious" web development at the start of the smartphone era, I learned JS throughout jQuery (It made much more sense at that time aswell as IE6, 7 and 8 were awful). Hence I'm not good at vanilla JS. But I am willing to do so.

In other words: there will be a non-jquery variant coming within the next few days.

**Comment by ghost** [comment]
Wow, that was fast! ;) Thank you, @andreruffert !

**Comment by lucidlemon** [comment]
Oh. This wasn't even you. I was just remembering Andre haha, well... good job by @andreruffert.
